tests/migration_test.go

```diff
@@ -3651,6 +3652,46 @@ var _ = Describe("[Serial][rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][leve
 			}, time.Second*30, time.Second*1).Should(BeNumerically("<=", 1))
 		})
 	})
+
+	Context("topology hints", func() {
+
+		Context("needs to be set when", func() {
+
+			expectTopologyHintsToBeSet := func(vmi *v1.VirtualMachineInstance) {
+				EventuallyWithOffset(1, func() bool {
+					vmi, err = virtClient.VirtualMachineInstance(vmi.Namespace).Get(vmi.Name, &metav1.GetOptions{})
+					Expect(err).ToNot(HaveOccurred())
+
+					return topology.AreTSCFrequencyTopologyHintsDefined(vmi)
+				}, 90*time.Second, 3*time.Second).Should(BeTrue(), fmt.Sprintf("tsc frequency topology hints are expected to exist for vmi %s", vmi.Name))
+			}
+
+			It("invtsc feature exists", func() {
+				vmi := libvmi.New("test-vmi",
```

Comment:
@iholder101 As #9222 got merged, you don't need to specify the VMI name in `libvmi.New("test-vmi",` anymore

Reply:
I don't understand how it even compiles :0
I thought that the bot would rebase automatically, let me rebase manually.